go.mod

```diff
@@ -8,9 +8,25 @@ replace github.com/google/cel-go => github.com/google/cel-go v0.16.1
 // This version taken from https://github.com/kubernetes/apiserver/blob/v0.28.4/go.mod#L50
 replace k8s.io/kube-openapi => k8s.io/kube-openapi v0.0.0-20230717233707-2695361300d9
 
+// Unfortuntely, fostite depends on ory/x which depends on very recent versions of opentelemetry.
+// We need to use the older versions of opentelemetry which are the direct dependencies of
+// https://github.com/kubernetes/apiserver/blob/v0.28.4/go.mod. Wherever they clash, replace
+// with the version which is more consistent with kubernetes/apiserver.
+replace (
+	go.opentelemetry.io/contrib/instrumentation/net/http/httptrace/otelhttptrace => go.opentelemetry.io/contrib/instrumentation/net/http/httptrace/otelhttptrace v0.35.0
```

Comment:
Yeah bummer to have to add all of this, but I guess it can't be helped. 😢

Reply:
Unfortunately, it will not compile without these replace directives.